### DISCUSSION TITLE
Configuration for IOT-GATE-iMX8 Yocto (Debian 10) linux

### DISCUSSION THREAD

**Discussion by aleh4d** [RaspAP/raspap-webgui#953] [General]
Need configuration for IOT-GATE-iMX8 device - Yocto (Debian 10) linux.
https://www.compulab.com/products/iot-gateways/iot-gate-imx8-industrial-arm-iot-gateway/
The device has WI-Fi, WAN and LAN ports.

**Comment by billz** [comment]
Search discussions for Yocto. At least one other user is working on this.

**Comment by aleh4d** [comment]
just maybe need a sample of configuration for the device with one WI-Fi, one WAN and one LAN ports,
The rest should be standard as Yocto is any way Debian 10 based.
